Get media bar working on mobile.

hworld opened the pull request.

hworld committed:
Beginning work to get media bar working on mobile.

niksudan committed:
Media bar items now scroll vertically on mobile

niksudan committed:
Move media bar controls height to separate variable file

niksudan commented:
Points to address:

- [x] **Cross Browser Testing** - [This makes use of `calc()`](https://github.com/gamejolt/frontend-lib/pull/42/files#diff-f1b90bbe9ae79abcd11ae44dea00799aR10), which isn't too great when it comes to browser support. Either look into not using it, or providing an acceptable fallback.
- [x] **Optimisation for Landscape Orientations** - The controls section blocks out 80px at the top of the screen, which is a lot more noticeable when viewing this in a landscape orientation. The lightbox needs to be able to somehow 'hide' the controls, or make them less intrusive.

niksudan committed:
Add support for mobile landscape orientation to media bars

Additionally add 10px margin right to lightbox controls on desktop to prevent hugging the side of the screen.

niksudan committed:
Add fallback for calc() usage

niksudan commented:
I've made it so the buttons appear in the bottom right when viewing it on mobile in landscape orientation.

This orientation adds padding at the bottom of the item regardless, so you can scroll down all the way and rest the controls in a place where it's not covering anything.

![image](https://user-images.githubusercontent.com/6004491/36948997-7b071e12-1fda-11e8-8e19-b1011399b4f7.png)

![image](https://user-images.githubusercontent.com/6004491/36948975-44284858-1fda-11e8-9f78-dc460e23c938.png)

hworld committed:
Merge branch 'master' into media-bar-mobile

hworld committed:
some minor tweaks
